xbmc/music/windows/MusicFileItemListModifier.cpp

```diff
@@ -67,7 +67,6 @@ void CMusicFileItemListModifier::AddQueuingFolder(CFileItemList& items)
     //  All album related nodes
   case NODE_TYPE_ALBUM:
     if (directoryNode->GetType() == NODE_TYPE_OVERVIEW) return;
-    break;
```

Comment:
I like to add the comment `// No break` so no one is confused later.

Reply:
maybe not hide the 'return' at the eol would also help :)

Reply:
As it is this change just removes the line that I added which broke things, restoring the code to how it had been for years seems the clearest fix. 

But Coverity won't like this bit of code again, and I am happy to rework this module to the prefered current layout standards and so that Coverity is happy too in a follow up PR.